osc-staging.py

```diff
@@ -395,9 +395,14 @@ def do_staging(self, subcmd, opts, *args):
             CheckDuplicateBinariesCommand(api).perform(opts.save)
         elif cmd == 'freeze':
             for prj in args[1:]:
-                prj = api.prj_from_letter(prj)
+                is_adi = False
+                if 'adi:' in prj:
+                    prj = api.adi_prj_from_adi_name(prj)
+                    is_adi = True
+                else:
+                    prj = api.prj_from_letter(prj)
                 print(Fore.YELLOW + prj)
-                FreezeCommand(api).perform(prj, copy_bootstrap=opts.bootstrap)
+                FreezeCommand(api).perform(prj, copy_bootstrap=opts.bootstrap, freeze_adi=is_adi)
```

Comment:
`freeze_adi` seems odd, as `prj` already indicates if `adi` or not and passing to `freeze` command which would imply one wants to freeze `prj`. I would drop the parameter and just check if `adi` inside freeze since the implementation of freeze differs based on project.